src/spell/modules/spell/spell/src/log.rs

```diff
@@ -0,0 +1,194 @@
+use eyre::WrapErr;
+use marine_rs_sdk::marine;
+use marine_sqlite_connector::State;
+
+use fluence_spell_dtos::value::UnitValue;
+
+use crate::auth::{is_by_creator, is_by_spell};
+use crate::schema::db;
+
+
+#[marine]
+pub struct AllLogsResult {
+    pub logs: Vec<String>,
+    pub success: bool,
+    pub error: String,
+}
+
+#[marine]
+pub fn store_log(log: String, timestamp: u64) -> UnitValue {
```

Comment:
Maybe get `timestamp` on insert into the table?
Smth like `instal into logs (timestamp, log) values (strftime('%s', 'now'), ?)`

Or even
```
sqlite> create table logs3 (log text, Timestamp INTEGER NOT NULL DEFAULT (strftime('%s', 'now')));
sqlite> insert into logs3 (log) values ('aaa');
sqlite> insert into logs3 (log) values ('aab');
sqlite> select * from logs3;
+-----+------------+
| log | Timestamp  |
+-----+------------+
| aaa | 1688735784 |
| aab | 1688735789 |
+-----+------------+
```